socket/packet/server_list_response.go

```diff
@@ -0,0 +1,60 @@
+package packet
+
+import (
+	"github.com/sandertv/gophertunnel/minecraft/protocol"
+)
+
+// ServerListResponse is sent by the proxy in response to ServerListRequest. It sends list of all
+// the servers connected to the proxy.
+type ServerListResponse struct {
+	// Servers represents all the servers connected to the proxy.
+	Servers []ServerEntry
+}
+
+// ServerEntry represents server connected the proxy.
+type ServerEntry struct {
+	// Name is name of the server.
+	Name string
+	// Group is group of the server.
+	Group string
+	// Online returns if the server is connected to the TCP socket server or not.
+	Online bool
+	// PlayerCount returns player count of the server.
+	PlayerCount uint16
```

Comment:
Is there any reason why this is an `uint16` instead of an `int64`?

Reply:
Player count could never be less than 0. Also I don't think more than 65535 players would join one server.

Reply:
Should still be consistent.